include/ofi_util.h

```diff
@@ -240,6 +240,8 @@ struct util_ep {
 	struct util_cq		*tx_cq;
 	uint64_t		tx_op_flags;
 	uint64_t		inject_op_flags;
+	uint64_t		tx_msg_flags;
+	uint64_t		rx_msg_flags;
```

Comment:
Can you add a comment that providers should OR this flag on *msg API calls if depending on util_ep for selective completion functionality?